### ISSUE TITLE
Package won't precompile due to deprecated `SceneOverlay` type

### ISSUE THREAD

**Issue by rbalexan** [sisl/AutomotiveSensors.jl#8]
Attempting to import the package into the workspace throws a precompilation error since `SceneOverlay` type isn't defined.

```
julia> using AutomotiveSensors
[ Info: Precompiling AutomotiveSensors [9ec66b5e-c605-11e8-1454-ede748df6f96]
ERROR: LoadError: LoadError: UndefVarError: SceneOverlay not defined
Stacktrace:
 [1] top-level scope at /Users/rossalexander/.julia/packages/Parameters/l76EM/src/Parameters.jl:570
 [2] include at ./boot.jl:328 [inlined]
 [3] include_relative(::Module, ::String) at ./loading.jl:1105
 [4] include at ./Base.jl:31 [inlined]
 [5] include(::String) at /Users/rossalexander/.julia/packages/AutomotiveSensors/kfhvu/src/AutomotiveSensors.jl:1
 [6] top-level scope at /Users/rossalexander/.julia/packages/AutomotiveSensors/kfhvu/src/AutomotiveSensors.jl:22
 [7] include at ./boot.jl:328 [inlined]
 [8] include_relative(::Module, ::String) at ./loading.jl:1105
 [9] include(::Module, ::String) at ./Base.jl:31
 [10] top-level scope at none:2
 [11] eval at ./boot.jl:330 [inlined]
 [12] eval(::Expr) at ./client.jl:425
 [13] top-level scope at ./none:3
in expression starting at /Users/rossalexander/.julia/packages/AutomotiveSensors/kfhvu/src/occlusion_checker.jl:36
in expression starting at /Users/rossalexander/.julia/packages/AutomotiveSensors/kfhvu/src/AutomotiveSensors.jl:22
ERROR: Failed to precompile AutomotiveSensors [9ec66b5e-c605-11e8-1454-ede748df6f96] to /Users/rossalexander/.julia/compiled/v1.3/AutomotiveSensors/NEFS3_5WWBd.ji.
Stacktrace:
 [1] compilecache(::Base.PkgId, ::String) at ./loading.jl:1283
 [2] _require(::Base.PkgId) at ./loading.jl:1024
 [3] require(::Base.PkgId) at ./loading.jl:922
 [4] require(::Module, ::Symbol) at ./loading.jl:917
```

It looks like `SceneOverlay` was removed from `AutoViz` (https://github.com/sisl/AutoViz.jl/pull/48). Obviously, switching over to `AutomotiveVisualization` altogether is preferred, but maybe there is a quick temporary fix? (I'm not too familiar with the AutoViz package yet)

Either:
- remove the subtype definition
- switch to a newer, appropriate supertype

**Comment by rbalexan** [comment]
I fixed a couple things in [autoviz-patch.zip](https://github.com/sisl/AutomotiveSensors.jl/files/4434565/autoviz-patch.zip) (I don't have push access to the repo to upload and create a PR). It should at least precompile now.

- `SceneOverlay` subtype definitions were removed (though one remains in the `test.ipynb` notebook)
 - `AutoViz.render!` --> `AutoViz.render`
- `AutomotiveDrivingModels.Vehicle` --> `AutomotiveDrivingModels.Entity`

**Comment by MaximeBouton** [comment]
I added you as a collaborator, I will open a PR today

**Comment by MaximeBouton** [comment]
#9

**Comment by rbalexan** [comment]
resolved by #9
